cms/static/js/views/tabs.js

```diff
@@ -0,0 +1,189 @@
+// Generated by CoffeeScript 1.6.1
+(function() {
+  var __hasProp = {}.hasOwnProperty,
+    __extends = function(child, parent) { for (var key in parent) { if (__hasProp.call(parent, key)) child[key] = parent[key]; } function ctor() { this.constructor = child; } ctor.prototype = parent.prototype; child.prototype = new ctor(); child.__super__ = parent.prototype; return child; };
+
+  define(["underscore", "jquery", "jquery.ui", "backbone", "common/js/components/views/feedback_prompt", "common/js/components/views/feedback_notification", "coffee/src/views/module_edit", "js/models/module_info", "js/utils/module"], function(_, $, ui, Backbone, PromptView, NotificationView, ModuleEditView, ModuleModel, ModuleUtils) {
+    var TabsEdit,
+      _this = this;
```

Comment:
can we remove this line (and line 14, and 24 and 64 etc) and search/replace the rest of the `_this` usages for `this`?